src/docbuilder/rustwide_builder.rs

```diff
@@ -215,7 +215,7 @@ impl RustwideBuilder {
                     let file_name = if versioned {
                         format!("{}-{}.{}", segments[1], rustc_version, segments[0])
                     } else {
-                        file.to_string()
+                        (*file).to_string()
```

Comment:
Redundant dereference
```suggestion
                        file.to_string()
```

Reply:
Same thing with the first one, it's more performant to dereference